### PR TITLE
feat(components): [select] :persistent="false" default value

### PR DESCRIPTION
**Pull request by BigPengZai**
fix [select] :persistent="false" default value

#6609

Please make sure these boxes are checked before submitting your PR, thank you!

- [ ] Make sure you follow contributing guide [English](https://github.com/element-plus/element-plus/blob/master/.github/CONTRIBUTING.en-US.md) | ([中文](https://github.com/element-plus/element-plus/blob/master/.github/CONTRIBUTING.zh-CN.md) | [Español](https://github.com/element-plus/element-plus/blob/master/.github/CONTRIBUTING.es.md) | [Français](https://github.com/element-plus/element-plus/blob/master/.github/CONTRIBUTING.fr-FR.md)).
- [ ] Make sure you are merging your commits to `dev` branch.
- [ ] Add some descriptions and refer to relative issues for your PR.

### CODE REVIEW

**Review by Jianger — CHANGES_REQUESTED** [review]
Thanks

**Review by Jianger — APPROVED** [review]
works

**Comment by msidolphin** [comment]
There is a problem with changes that return persistentRef, this will make `shouldRender ` always true

File: packages/components/tooltip/src/content.vue [hunk 1]
```diff
@@ -102,7 +102,7 @@ export default defineComponent({
     })
 
     const shouldRender = computed(() => {
-      return unref(persistentRef) ? true : unref(open)
+      return persistentRef
```
Comment (msidolphin) [inline packages/components/tooltip/src/content.vue:105]: There is a problem with changes that return persistentRef, this will make shouldRender always true

File: packages/components/menu/src/menu-item.vue [hunk 2]
```diff
@@ -19,6 +19,7 @@
       :effect="Effect.DARK"
       placement="right"
       persistent
+      :fallback-placements="['right', 'left', 'bottom', 'top']"
```
Comment (msidolphin) [inline packages/components/menu/src/menu-item.vue:22]: remove unrelated modifications

**Comment by msidolphin** [comment]
Still not right

File: packages/components/menu/src/menu-item.vue (hunk 2 above)
Comment (BigPengZai) [reply packages/components/menu/src/menu-item.vue]: :fallback-placements="['right', 'left']"

File: packages/components/tooltip/src/content.vue (hunk 1 above)
Comment (BigPengZai) [reply packages/components/tooltip/src/content.vue]: return unref(persistentRef) ? true : ref(open)

File: packages/components/menu/src/menu-item.vue (hunk 2 above)
Comment (BigPengZai) [reply packages/components/menu/src/menu-item.vue]: I'm not sure. Please point it out

**Comment by BigPengZai** [comment]
> Still not right

I'm not sure. Please point it out

**Comment by msidolphin** [comment]
> > Still not right
> 
> I'm not sure. Please point it out

when persistent is false, select dropdown will be destroyed, your changes will make the persistent property not work

File: packages/components/menu/src/menu-item.vue (hunk 2 above)
Comment (msidolphin) [reply packages/components/menu/src/menu-item.vue]: > I'm not sure. Please point it out

I don't understand what this change has to do with `select`

Comment (BigPengZai) [reply packages/components/menu/src/menu-item.vue]: > remove unrelated modifications

this pr ，fix #6632

**Comment by BigPengZai** [comment]
resubmit
